src/server/schema.ts

```diff
@@ -30,11 +30,30 @@ export const schema = `
 
   type Terms {
     source: Source!
-    terms: [Term]!
+    terms: [Term]! @deprecated(reason: "Use 'result' instead")
```

Comment:
Kept this for BC, but deprecated it to point clients to the new `result` property.

Reply:
👍🏻

Reply:
By the way, an alternative approach would be to keep `terms` (and don’t deprecate it) and _add_ an `errors` list property (that can be empty) instead of using a union type.